.github/workflows/publish.yml

```diff
@@ -16,15 +17,24 @@ jobs:
         with:
           node-version: 14
           cache: 'yarn'
-          registry-url: 'https://npm.pkg.github.com'
+          registry-url: 'https://registry.npmjs.org'
 
       - name: Yarn Install
         run: yarn install --network-concurrency 1
 
       - name: Build
         run: yarn prod
 
-      - name: Publish
+      - name: Publish (NPM)
+        run: npm publish
+        env:
+          NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
+
+      - uses: actions/setup-node@v2
+        with:
+          registry-url: 'https://npm.pkg.github.com'
```

Comment:
Do you not need the `node-version` or `cache` in this one?

Reply:
Negative, only the first time. Just swapping registry on 2nd.